extensions/ql-vscode/package.json

```
@@ -289,6 +289,10 @@
         "command": "codeQL.runRemoteQuery",
         "title": "CodeQL: Run Remote Query"
       },
+      {
+        "command": "codeQL.showFakeRemoteQueryResults",
+        "title": "CodeQL: Show fake remote query results"
```

Would be nice to add some indication that this is for extension dev work only. Maybe something like:

```suggestion
        "title": "[Internal] CodeQL: Show fake remote query results"
```

But, you can come up with your own label. It's nice that it only shows for canary users, but many of our internal users are canary users. It would be nice not to confuse them.